analysis/vm-script-caching-investigation-2026-06-07.md

```
@@ -0,0 +1,195 @@
+- Add code complexity (cache invalidation, memory management)
+- Provide <0.01% improvement on real workloads
+- Create false confidence that "we optimized the VM layer"
+
+## Artifacts
+
+Benchmark scripts and raw data available at:
+- Original experiment: `vm-script-cache/tmp/experiments/20260607T083551Z-vm-script-cache/`
+
+### Raw Benchmark Output (Summary)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)